Code/EnginePlugins/FmodPlugin/Components/FmodEventComponent.cpp

```diff
@@ -652,27 +652,27 @@ ezInt32 ezFmodEventComponent::FindParameter(const char* szName) const
     return -1;
 
   FMOD_STUDIO_PARAMETER_DESCRIPTION paramDesc;
-  if (pEventDesc->getParameter(szName, &paramDesc) != FMOD_OK)
+  if (pEventDesc->getParameterDescriptionByName(szName, &paramDesc) != FMOD_OK)
     return -1;
 
-  return paramDesc.index;
+  return *reinterpret_cast<ezFmodParameterId*>(&paramDesc.id);
 }
 
-void ezFmodEventComponent::SetParameter(ezInt32 iParamIndex, float fValue)
+void ezFmodEventComponent::SetParameter(ezFmodParameterId ParamId, float fValue)
 {
-  if (m_pEventInstance == nullptr || iParamIndex < 0 || !m_pEventInstance->isValid())
+  if (m_pEventInstance == nullptr || !m_pEventInstance->isValid() || ParamId == (ezFmodParameterId)-1)
```

Comment:
Isn't there some invalid id define in fmod maybe?

Reply:
Unfortunately not